var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -33,17 +33,19 @@ class Openssl(Package):
        Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.
        It is also a general-purpose cryptography library."""
     homepage = "http://www.openssl.org"
-    url      = "http://www.openssl.org/source/openssl-1.0.1h.tar.gz"
-    list_url = "https://www.openssl.org/source/old/"
+    url = "http://www.openssl.org/source/openssl-1.0.1h.tar.gz"
+    list_url = "http://www.openssl.org/source/old/"
```

Comment:
With `http`, `spack versions` and `spack checksum` no longer work. Is there a reason you made this change?